server/image_status.go

```diff
@@ -63,7 +65,13 @@ func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (r
 				RepoDigests: status.RepoDigests,
 				Size_:       size,
 			},
-			Info: status.Labels,
```

Comment:
`Info` was always documented to be set only if `req.Verbose`, but is it OK at this point in the release to remove it?

Also, the way `Info` now represents labels has changed. Is it OK to change that?

Reply:
Yes I think It is okay since we did not use `Info` at all before 1.18. :+1: